frontend/apps/desktop/src/models/daemon.ts

```diff
@@ -111,6 +108,33 @@ export function useMyAccounts() {
   return useEntities(data?.map((k) => hmId('d', k)))
 }
 
+/**
+ * Returns a list of keys from the daemon.
+ * This is a hook that is used to list keys from the daemon.
+ * It is used to check if the user has any accounts.
+ * If the user has no accounts, it will show the onboarding screen.
+ * If the user has accounts, it will show the main app.
+ *
+ * @returns
+ */
+export function useListKeys() {
```

Comment:
whats wrong with useMyAccountIds ? Why not use react query?

Reply:
because I want as minimal things as possible in the main process. I don't want to deal with react query stuff in there

Reply:
oh! Wait yeah this is the main process. So why does it have React useEffect? How does this even work?

Reply:
what do you mean, in the main process? This is being used in the frontend. We have already loaded react-query at this time. Can we please just `useMyAccountIds` like we normally do?